test/NuGet.Clients.Tests/NuGet.CommandLine.Test/NuGetListCommandTest.cs

```diff
@@ -18,6 +20,9 @@ namespace NuGet.CommandLine.Test
 {
     public class NuGetListCommandTest
     {
```

Comment:
Why do we need list command tests for the source command?>

Reply:
I did a rebase on dev-feature-http-error. The rebase led to the dev-feature-http-error losing the list command updates I am not sure why. As a result, it ended up looking like the updates for the list command were made on this PR. I did a rest on the feature branch and recreated this PR

Reply:
https://github.com/NuGet/NuGet.Client/pull/5702